### DISCUSSION TITLE
Timeout not working

### DISCUSSION THREAD

**Discussion by Zach-Meadows** [TwitchIO/TwitchIO#386] [Questions] [answered]
previously i was able to just do ctx.send("/timeout {name}") and it worked fine but now it doesn't seem to work. I've also used the User.send_timeout function. but to no avail. Help please!

**Comment by chillymosh** [comment, marked as answer]
@Zach-Meadows Twitch no longer support chat commands via IRC and therefore bots. You will need to use the API endpoints now with the correct scopes on a user oauth token.

**Reply by Zach-Meadows** [reply]
Thank you! I was pulling my hair out.
